modules/service/src/main/scala/lucuma/odb/data/Nullable.scala

```diff
@@ -23,6 +23,15 @@ sealed trait Nullable[+A] extends Product with Serializable {
   def orElse[B >: A](nb: Nullable[B]): Nullable[B] = fold(nb, nb, NonNull(_))
   def toOption: Option[A] = fold(None, None, Some(_))
 
+  def isNull: Boolean =
+    fold(ifNull = true, ifAbsent = false, _ => false)
+
+  def isAbsent: Boolean =
+    fold(ifNull = false, ifAbsent = true, _ => false)
+
+  def isPresent: Boolean =
+    fold(ifNull = false, ifAbsent = false, _ => true)
```

Comment:
I didn't end up using these or the `orNull` or `orAbsent` constructors.  We can delete them if it meets opposition.